Lab2/src/UI/View/CrudConsoleService/GroupCrudConsoleService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using BLL.Models;
+using BLL.Services;
+using System;
+using UI.Interfaces;
+
+namespace UI.View.CrudConsoleService
```

Comment:
View?
Services - better folder and withoout additional `CrudConsoleService` folder

Reply:
Fixed